symbolic-symcache/tests/test_writer.rs

```diff
@@ -106,7 +92,7 @@ fn test_write_header_macos() -> Result<(), Error> {
         },
         arch: Amd64,
         files: 36,
-        functions: 639,
+        functions: 587,
```

Comment:
Likewise, `test_write_header_macos` and `test_write_functions_macos` also source their data from the same file. Here's a much longer list covering all 52 missing functions:
```
ffffffff _ZN15google_breakpad10scoped_ptrI9sigactionED1Ev
ffffffff _ZN15google_breakpad10scoped_ptrINS_21CrashGenerationClientEED1Ev
ffffffff _ZN15google_breakpad11MachMessage13CalculateSizeEv
ffffffff _ZN15google_breakpad11MachMessage13GetDataPacketEv
ffffffff _ZN15google_breakpad11MachMessage13GetDescriptorEi
ffffffff _ZN15google_breakpad11MachMessage13SetDescriptorEiRKNS_21MachMsgPortDescriptorE
ffffffff _ZN15google_breakpad11MachMessage18SetDescriptorCountEi
ffffffff _ZN15google_breakpad11MachMessage7SetDataEPvi
ffffffff _ZN15google_breakpad11MachMessageC2Ev
ffffffff _ZN15google_breakpad11ReceivePortC2EPKc
ffffffff _ZN15google_breakpad11ReceivePortC2Ej
ffffffff _ZN15google_breakpad11ReceivePortC2Ev
ffffffff _ZN15google_breakpad11ReceivePortD2Ev
ffffffff _ZN15google_breakpad12DynamicImage11GetFileTypeEv
ffffffff _ZN15google_breakpad13DynamicImages20ReadImageInfoForTaskEv
ffffffff _ZN15google_breakpad13DynamicImages23GetExecutableImageIndexEv
ffffffff _ZN15google_breakpad13DynamicImages27GetDyldAllImageInfosPointerEv
ffffffff _ZN15google_breakpad14MachPortSenderC2EPKc
ffffffff _ZN15google_breakpad14MachPortSenderC2Ej
ffffffff _ZN15google_breakpad15MachSendMessageC2Ei
ffffffff _ZN15google_breakpad16ExceptionHandler13ResumeThreadsEv
ffffffff _ZN15google_breakpad16ExceptionHandler14SuspendThreadsEv
ffffffff _ZN15google_breakpad16ExceptionHandler26SendMessageToHandlerThreadENS_20HandlerThreadMessageE
ffffffff _ZN15google_breakpad16ExceptionHandler8TeardownEv
ffffffff _ZN15google_breakpad16ExceptionHandlerC1ERKNSt3__112basic_stringIcNS1_11char_traitsIcEENS1_9allocatorIcEEEEPFbPvEPFbPKcSE_SA_bESA_bSE_
ffffffff _ZN15google_breakpad16ExceptionHandlerD1Ev
ffffffff _ZN15google_breakpad17MinidumpGenerator10WriteStackEPjP18MDMemoryDescriptor
ffffffff _ZN15google_breakpad17MinidumpGenerator12WriteContextEPjP20MDLocationDescriptor
ffffffff _ZN15google_breakpad17MinidumpGenerator13WriteStackX86EPjP18MDMemoryDescriptor
ffffffff _ZN15google_breakpad17MinidumpGenerator14GetThreadStateEjPjS1_
ffffffff _ZN15google_breakpad17MinidumpGenerator16WriteStackX86_64EPjP18MDMemoryDescriptor
ffffffff _ZN15google_breakpad17MinidumpGenerator17CurrentPCForStackEPj
ffffffff _ZN15google_breakpad17MinidumpGenerator20CurrentPCForStackX86EPj
ffffffff _ZN15google_breakpad17MinidumpGenerator20FindExecutableModuleEv
ffffffff _ZN15google_breakpad17MinidumpGenerator23CurrentPCForStackX86_64EPj
ffffffff _ZN15google_breakpad17MinidumpGeneratorD1Ev
ffffffff _ZN15google_breakpad18MinidumpFileWriter4CopyEjPKvl
ffffffff _ZN15google_breakpad18MinidumpFileWriter5CloseEv
ffffffff _ZN15google_breakpad18MinidumpFileWriter8AllocateEm
ffffffff _ZN15google_breakpad18MinidumpFileWriterC2Ev
ffffffff _ZN15google_breakpad18MinidumpFileWriterD2Ev
ffffffff _ZN15google_breakpad19breakpad_exc_serverEP17mach_msg_header_tS1_
ffffffff _ZN15google_breakpad6FileIDC2EPKc
ffffffff _ZN16MacFileUtilities11MachoWalker20WalkHeader64AtOffsetEx
ffffffff _ZN16MacFileUtilities11MachoWalker9ReadBytesEPvmx
ffffffff _ZN16MacFileUtilities11MachoWalkerC2EPKcPFbPS0_P12load_commandxbPvES6_
ffffffff _ZN16MacFileUtilities11MachoWalkerC2EPvmPFbPS0_P12load_commandxbS1_ES1_
ffffffff _ZN16MacFileUtilities11MachoWalkerD2Ev
ffffffff _ZN16MacFileUtilities7MachoID6UpdateEPNS_11MachoWalkerExm
ffffffff _ZN16MacFileUtilities7MachoIDC2EPKc
ffffffff _ZN16MacFileUtilities7MachoIDC2EPKcPvm
ffffffff _ZNSt3__114__split_bufferI18MDMemoryDescriptorRN15google_breakpad16PageStdAllocatorIS1_EEEC2EmmS5_
```

If you'd like to generate these yourself, check out `master`, comment out L22-37 of this file and insert a `Some(f)` above or below that block, and run the tests for this crate. insta will generate a new snapshot based on your test results which will contain all of the functions in the symcache, including the inlined (duplicate and non-duplicate) functions.